### PR TITLE
Use constants for heal locations

### PR DESCRIPTION
**Pull request by garakmon**

### CODE REVIEW

File: src/region_map.c [hunk 1]
```diff
@@ -1260,58 +1262,59 @@ static const u8 sFlyRegionMapFrame_TilemapLZ[] = INCBIN_U8("graphics/pokenav/map
...
-    {0, 44,  0},
-    {0, 45,  0},
-    {0, 46,  0},
-    {0, 47,  0},
-    {0, 48,  0},
-    {0, 49,  0},
+// MAP_GROUP, MAP_NUM, HEAL_LOCATION
+static const u8 sMapHealLocationIndexes[][3] =
+{
+/* MAPSEC_LITTLEROOT_TOWN  */    {0,  9, HEAL_LOCATION_LITTLEROOT_TOWN_BRENDANS_HOUSE_2F},
```
Comment (huderlem) [inline src/region_map.c]: Use the constants for map group and map num.
```
{MAP_GROUP(OLDALE_TOWN), MAP_NUM(OLDALE_TOWN), HEAL_LOCATION_OLDALE_TOWN},
```

File: src/region_map.c [hunk 2]
```diff
@@ -1260,58 +1262,59 @@ static const u8 sFlyRegionMapFrame_TilemapLZ[] = INCBIN_U8("graphics/pokenav/map
...
-    {0, 42,  0},
-    {0, 43,  0},
-    {0, 44,  0},
-    {0, 45,  0},
-    {0, 46,  0},
-    {0, 47,  0},
-    {0, 48,  0},
-    {0, 49,  0},
+// MAP_GROUP, MAP_NUM, HEAL_LOCATION
+static const u8 sMapHealLocationIndexes[][3] =
```
Comment (huderlem) [inline src/region_map.c]: Just name this `sMapHealLocations`

File: src/region_map.c [hunk 3]
```diff
@@ -1333,6 +1336,7 @@ static const struct MultiPartMapSection sMultiPartMapSections[1] =
     {sEverGrandeCityAreaNames, MAPSEC_EVER_GRANDE_CITY, FLAG_SYS_POKEMON_LEAGUE_FLY},
 };
 
+// what in the hell is this?
```
Comment (huderlem) [inline src/region_map.c]: either remove the comment or figure out a proper name for `UnknownStruct3`

File: src/region_map.c (hunk 1 above)
Comment (huderlem) [reply src/region_map.c]: And then all of the inline `MAPSEC` comments can be removed

File: src/region_map.c (hunk 3 above)
Comment (garakmon) [reply src/region_map.c]: oops

**Comment by garakmon** [comment]
fixed

File: src/region_map.c [hunk 4]
```diff
@@ -1260,58 +1262,58 @@ static const u8 sFlyRegionMapFrame_TilemapLZ[] = INCBIN_U8("graphics/pokenav/map
...
-    {0, 43,  0},
-    {0, 44,  0},
-    {0, 45,  0},
-    {0, 46,  0},
-    {0, 47,  0},
-    {0, 48,  0},
-    {0, 49,  0},
+static const u8 sMapHealLocations[][3] =
+{
+/* MAPSEC_LITTLEROOT_TOWN  */    {MAP_GROUP(LITTLEROOT_TOWN), MAP_NUM(LITTLEROOT_TOWN), HEAL_LOCATION_LITTLEROOT_TOWN_BRENDANS_HOUSE_2F},
```
Comment (huderlem) [inline src/region_map.c]: These comments should be removed now that the constants are being used.

**Pull request merged**